website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -764,26 +768,27 @@ Views with this configuration will be able to select from objects in `project_1.
 
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
 
-<VersionBlock firstVersion="1.6">
-
+<VersionBlock firstVersion="1.7">
 
 ## Materialized view
 
 The BigQuery adapter supports [materialized views](https://cloud.google.com/bigquery/docs/materialized-views-intro) and refreshes them for every subsequent `dbt run` you execute. For more information, see [Refresh Materialized Views](https://cloud.google.com/bigquery/docs/materialized-views-manage#refresh) in the Google docs.
 
-Materialized views support the optional configuration `on_configuration_change` with the following values: 
+Materialized views support the optional configuration `on_configuration_change` with the following values:
+
 - `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The following changes can be applied without the need to rebuild the materialized view:
-  - enable_refresh 
-  - refresh_interval_minutes 
-  - max_staleness
+    - enable_refresh
+    - refresh_interval_minutes
+    - max_staleness
 - `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
```

Comment:
This is called `continue` now. We abandoned `skip` because that's associated with another status in dbt and we felt it was too close and would cause confusion.